src/components/OrgListCard/OrgListCard.tsx

```
@@ -95,16 +97,16 @@
             </Tooltip>
             {/* Description of the organization */}
             <h6 className={`${styles.orgdesc} fw-semibold`}>
+              {/* <TruncatedText text={userData?.organizations[0].description || ''} /> */}
               <span>{userData?.organizations[0].description}</span>
             </h6>
+
             {/* Display the organization address if available */}
             {address && address.city && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)